packages/rest/src/validation/request-body.validator.ts

```diff
@@ -33,7 +33,7 @@ const ajvErrors = require('ajv-errors');
  * @param globalSchemas - The referenced schemas generated from `OpenAPISpec.components.schemas`.
  * @param options - Request body validation options for AJV
  */
-export function validateRequestBody(
+export async function validateRequestBody(
```

Comment:
Should this be considered a breaking change?

Reply:
Yes, it should be considered a breaking change.

Reference: https://github.com/strongloop/loopback-next/pull/4658#discussion_r382607886

Reply:
@mastermunj Could we amend the commit message to include the `BREAKING CHANGE` flag? This can be done with `git commit --amend`

Suggested commit message:
```
feat(rest): add async validation support

BREAKING CHANGE: `validateRequestBody` is now async

feat(rest): simplified ajv validation promise use

feat(rest): one more improvement
```

This'll ensure a semver major bump.

Reply:
@achrinza, I have amended the commit message as you suggested.